app/models/patient.rb

```diff
@@ -252,6 +255,11 @@ def has_special_circumstances
     !!has_circumstance
   end
 
+  # check if a patient is long past waiting for audit
+  def obsolete?
```

Comment:
love this pattern!

we can simplify the code itself. `initial_call_date < 2.years.ago` should produce the same result without needing the `self` or the ternary operator.